ecommerce/admin.py

```diff
@@ -156,7 +156,62 @@ def order_link(self, obj):
     order_link.short_description = "Order"
 
 
+class WireTransferReceiptAdmin(TimestampedModelAdmin):
+    """Admin for WireTransferReceipt"""
+
+    model = WireTransferReceipt
+    include_created_on_in_list = True
+    readonly_fields = get_field_names(WireTransferReceipt) + ["order_link"]
+    list_display = ("id", "get_user_email", "order_link", "get_order_status")
+    search_fields = ("order__user__email", "order__user__username")
+
+    def has_add_permission(self, request):
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        return False
+
```

Comment:
Since there aren't any editable fields, maybe add this to remove the `Save` buttons:

```python
def has_change_permission(self, request, obj=None):
        return False
```